docs/src/modules/components/AppLayoutDocs.js

```diff
@@ -37,12 +38,7 @@ const useStyles = makeStyles((theme) => ({
     },
   },
   toc: {
-    [theme.breakpoints.up('sm')]: {
-      width: 'calc(100% - 175px)',
-    },
-    [theme.breakpoints.up('lg')]: {
-      width: 'calc(100% - 175px - 240px)',
-    },
+    width: 'calc(100% - 175px)',
```

Comment:
I consider this a win as well. The component is now coupled less to the app drawer (width).